### PR TITLE
Update coreos-ncp profile and map AU-12(1), AC-12, and AC-2(5)

### LINKED ISSUE #4185: CoreOS: AU-12(1)	AUDIT GENERATION | SYSTEM-WIDE / TIME-CORRELATED AUDIT TRAIL
https://nvd.nist.gov/800-53/Rev4/control/AU-12?baseline=high#enhancement-1

AU-12(1) | AUDIT GENERATION \| SYSTEM-WIDE / TIME-CORRELATED AUDIT TRAILThe information system compiles audit records from [Assignment: organization-defined information system components] into a system-wide (logical or physical) audit trail that is time-correlated to within [Assignment: organization-defined level of tolerance for the relationship between time stamps of individual records in the audit trail].Supplemental Guidance: Audit trails are time-correlated if the time stamps in the individual audit records can be reliably related to the time stamps in other audit records to achieve a time ordering of the records within organizational tolerances. Related to: AU-8, AU-12
-- | --

### PR DESCRIPTION
**Pull request by redhatrises**
- Fixes #4185

### CODE REVIEW

File: linux_os/guide/services/ntp/chronyd_or_ntpd_specify_remote_server/rule.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ references:
     cis: "3.6"
     cui: 3.3.7
     disa: "160,1891"
-    nist: AU-8(1),AU-8(1)(a)
+    nist: AU-8(1),AU-8(1)(a),AU-12(1)
```
Comment (shawndwells) [inline linux_os/guide/services/ntp/chronyd_or_ntpd_specify_remote_server/rule.yml]:
```suggestion
    nist: AU-8(1)(a)
```

Comment (shawndwells) [reply linux_os/guide/services/ntp/chronyd_or_ntpd_specify_remote_server/rule.yml]: AU-8 (1)(a) | Compares the internal information system clocks [Assignment: organization-defined frequency] with [Assignment: organization-defined authoritative time source]; and
-- | --

Comment (shawndwells) [reply linux_os/guide/services/ntp/chronyd_or_ntpd_specify_remote_server/rule.yml]: https://nvd.nist.gov/800-53/Rev4/control/AU-8#enhancement-1

File: linux_os/guide/services/ntp/ntpd_specify_multiple_servers/rule.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ identifiers:
     cce@rhel6: 26958-9
 
 references:
-    nist: AU-8(1)
+    nist: AU-8(1),AU-12(1)
```
Comment (shawndwells) [inline linux_os/guide/services/ntp/ntpd_specify_multiple_servers/rule.yml]: Believe there is a more specific NIST control about multiple time servers.... but having trouble finding it.

File: linux_os/guide/services/ntp/ntpd_specify_remote_server/rule.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ identifiers:
 
 references:
     disa@rhel6: "160"
-    nist: AU-8(1)
+    nist: AU-8(1),AU-12(1)
```
Comment (shawndwells) [inline linux_os/guide/services/ntp/ntpd_specify_remote_server/rule.yml]:
```suggestion
    nist: AU-8(1)(a)
```

File: linux_os/guide/services/ntp/package_ntp_installed/rule.yml [hunk 4]
```diff
@@ -11,7 +11,7 @@ severity: high
 references:
     anssi: NT012(R03)
     disa: "160"
-    nist: AU-8(1)
+    nist: AU-8(1),AU-12(1)
```
Comment (shawndwells) [inline linux_os/guide/services/ntp/package_ntp_installed/rule.yml]:
```suggestion
    nist: CM-6(a),CM-6(b)
```

Comment (shawndwells) [reply linux_os/guide/services/ntp/package_ntp_installed/rule.yml]: This one is hard. AU-8(1)(a) and AU-8(1)(b) need a time server, but the NIST controls are not specific enough to map to installing packages.

What if we use CM-6(a) and CM-6(b), which is kind of a catch-all "create and implement configuration settings" requirement?

File: linux_os/guide/services/ntp/service_chronyd_or_ntpd_enabled/rule.yml [hunk 5]
```diff
@@ -43,7 +43,7 @@ references:
     cis: 2.2.1.1
     cui: 3.3.7
     disa: "160"
-    nist: AU-8(1)
+    nist: AU-8(1),AU-12(1)
```
Comment (shawndwells) [inline linux_os/guide/services/ntp/service_chronyd_or_ntpd_enabled/rule.yml]:
```suggestion
    nist: CM-6(a),CM-6(b)
```

File: linux_os/guide/services/ntp/service_ntp_enabled/rule.yml [hunk 6]
```diff
@@ -22,7 +22,7 @@ severity: high
 references:
     anssi: NT012(R03)
     disa: "160"
-    nist: AU-8(1)
+    nist: AU-8(1),AU-12(1)
```
Comment (shawndwells) [inline linux_os/guide/services/ntp/service_ntp_enabled/rule.yml]:
```suggestion
    nist: CM-6(a),CM-6(b)
```

File: linux_os/guide/services/ntp/service_ntpd_enabled/rule.yml [hunk 7]
```diff
@@ -24,7 +24,7 @@ identifiers:
 
 references:
     disa@rhel6: "160"
-    nist: AU-8(1)
+    nist: AU-8(1),AU-12(1)
```
Comment (shawndwells) [inline linux_os/guide/services/ntp/service_ntpd_enabled/rule.yml]:
```suggestion
    nist: CM-6(a),CM-6(b)
```

File: linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml [hunk 8]
```diff
@@ -26,7 +26,7 @@ identifiers:
 references:
     cui: 3.1.11
     disa: 1133,0361
-    nist: AC-12,SC-10
+    nist: AC-12,SC-10,AC-2(5)
```
Comment (shawndwells) [inline linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml]: ack

File: ocp4/profiles/coreos-ncp.profile [hunk 9]
```diff
@@ -189,16 +189,13 @@ selections:
     ### Install Required Packages
     - package_sssd-ipa_installed
     - package_aide_installed
-    #- package_dnf-automatic_installed
```
Comment (shawndwells) [inline ocp4/profiles/coreos-ncp.profile]: Needed for auto apply security patches

File: ocp4/profiles/coreos-ncp.profile [hunk 10]
```diff
@@ -189,16 +189,13 @@ selections:
...
     - package_firewalld_installed
     - package_iptables_installed
     - package_libcap-ng-utils_installed
-    #- package_openscap-scanner_installed
```
Comment (shawndwells) [inline ocp4/profiles/coreos-ncp.profile]: Needed to config scan the system

File: ocp4/profiles/coreos-ncp.profile [hunk 11]
```diff
@@ -579,3 +576,6 @@ selections:
     # AU-11
     - auditd_data_retention_num_logs
     - auditd_data_retention_max_log_file
+
+    # AC-2(5), AC-12
+    - accounts_tmout
```
Comment (shawndwells) [inline ocp4/profiles/coreos-ncp.profile]: Is a variable also needed?

File: linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/rule.yml [hunk 12]
```diff
@@ -26,7 +26,7 @@ identifiers:
 
 references:
     disa: 1891,2046
-    nist: AU-8(1)(a)
+    nist: AU-8(1)(a),AU-12(1)
```
[the remark this answers is not in the document]
Comment (redhatrises) [reply linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/rule.yml]: @shawndwells what maps to AU-12(1) then? auditd running?

File: ocp4/profiles/coreos-ncp.profile (hunk 9 above)
Comment (redhatrises) [reply ocp4/profiles/coreos-ncp.profile]: @shawndwells this is a brand new rule entirely as you cannot use yum/dnf.

File: ocp4/profiles/coreos-ncp.profile (hunk 11 above)
Comment (redhatrises) [reply ocp4/profiles/coreos-ncp.profile]: @shawndwells it should use the default variable assigned which is 10 minutes.

File: ocp4/profiles/coreos-ncp.profile (hunk 10 above)
Comment (redhatrises) [reply ocp4/profiles/coreos-ncp.profile]: @shawndwells this is a brand new rule entirely as you cannot use yum/dnf.

File: linux_os/guide/services/ntp/service_ntpd_enabled/rule.yml (hunk 7 above)
Comment (redhatrises) [reply linux_os/guide/services/ntp/service_ntpd_enabled/rule.yml]: Updating to CM is out of scope of this PR.

File: linux_os/guide/services/ntp/service_ntp_enabled/rule.yml (hunk 6 above)
Comment (redhatrises) [reply linux_os/guide/services/ntp/service_ntp_enabled/rule.yml]: Updating to CM is out of scope of this PR.

File: linux_os/guide/services/ntp/service_chronyd_or_ntpd_enabled/rule.yml (hunk 5 above)
Comment (redhatrises) [reply linux_os/guide/services/ntp/service_chronyd_or_ntpd_enabled/rule.yml]: Updating to CM is out of scope of this PR.

File: linux_os/guide/services/ntp/package_ntp_installed/rule.yml (hunk 4 above)
Comment (redhatrises) [reply linux_os/guide/services/ntp/package_ntp_installed/rule.yml]: Updating to CM is out of scope of this PR.

File: ocp4/profiles/coreos-ncp.profile (hunk 10 above)
Comment (redhatrises) [reply ocp4/profiles/coreos-ncp.profile]: Added the rule for discussion later

File: linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/rule.yml (hunk 12 above)
[the remark this answers is not in the document]
Comment (redhatrises) [reply linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/rule.yml]: Talked to @shawndwells and have mapped to rules where auditd is running.

**Pull request merged**